caps/templates/caps/council_cards/council-documents.html

```diff
@@ -0,0 +1,130 @@
+                                                {% include 'caps/icons/file.html' with width='3rem' height='3rem' classes='mr-3 flex-grow-0 flex-shrink-0' role='presentation' %}
+                                            {% endif %}
+                                            <div>
+                                                {% if plandocument.title %}
+                                                    <strong class="d-block">{{ plandocument.title|document_title }}</strong>
+                                                {% else %}
+                                                    <strong class="d-block">{{ council.name }} {{ plandocument.get_document_type|document_title }}</strong>
+                                                {% endif %}
+                                                <p class="d-inline-block text-muted mb-0">
+                                                    {% if plandocument.title %}
+                                                        {{ plandocument.get_document_type|title }}
+                                                        <span class="mx-2">&middot;</span>
+                                                    {% endif %}
+                                                    {% if plandocument.file_type %}{{ plandocument.file_type|upper }}{% endif %}
+                                                    {% if plandocument.get_status %}
+                                                        <span class="mx-2">&middot;</span>
+                                                        {{ plandocument.get_status|title }}
+                                                    {% endif %}
+                                                    {% if plandocument.get_scope %}
+                                                        <span class="mx-2">&middot;</span>
+                                                        {{ plandocument.get_scope|title }}
+                                                    {% endif %}
+                                                    {% if plandocument.is_deleted and not plandocument.file_exists %}
+                                                        <span class="mx-2">&middot;</span>
+                                                        We no longer hold a copy of this document
+                                                    {% endif %}
+                                                </p>
+                                            </div>
+                                            <div></div>
+                                            {% if plandocument.is_deleted %}
+                                                {% if plandocument.file_exists %}</a>{% endif %}
+                                            {% else %}
+                                            </a>
+                                        </div>
+                                        {% if plandocument.sorted_key_terms %}
```

Comment:
Is there a reason you think we should include this list of terms below each document?

From a quick test here, it looks like most documents have at least a few dozen terms in them, which makes this "wall" of tags pretty difficult to parse or make use of.

I think I’d either:

1. Show only the most frequent terms in each document (along with the number of times they appear) – although, TBH, I’m still not sure how useful this will be in reality.
2. Remove this listing of terms entirely, but pull out the most frequent terms across _all documents_ for the council, and present them as search suggestions under the search box:

<img width="795" alt="Screenshot 2023-04-04 at 11 23 15" src="https://user-images.githubusercontent.com/739624/229763327-05579ec8-accd-4a9a-ab91-5bf6c8dd2f64.png">

In either case, we have to set a cut-off somewhere for how many tags we want to display, because I really don’t think more than a handful will make sense.

The other alternative, of course, is that we just remove these bits _entirely_ from the council detail page, and use them _only_ for semantic search on the search page.

Reply:
So I think this is actually the really cool feature I think - possibly more useful that the search improvement itself.

It's not so much the little tags, but the fact you can click through and the search is letting you see the relevant bit inside that document. This is why a reasonable amount of tags isn't necessarily bad (but probably fewer than they are), because it's all jumping off points to inside the documents. 

Basically, this is making it easier for visitors to more quickly jump into bits of a plan, rather than hitting a 'now download this pdf, then look inside the pdf' wall.